### ISSUE TITLE
LearningCoreAudioWithSwift3.0

### ISSUE THREAD

**Issue by Necktwi** [AlesTsurko/LearningCoreAudioWithSwift2.0#3]
Would you mind starting your new repo on LearningCoreAudioWith3.0 with playback example first?
Thanks a lot!

**Comment by ales-tsurko** [comment]
Hi,
I have no any plans to fork/update this repository, sorry. But I'll keep your request opened, maybe someone would like to make it.

**Comment by dimagimburg** [comment]
@AlesTsurko Hi, can I contribute to this repository and convert few of the code examples to Swift 3?

**Comment by ales-tsurko** [comment]
@dimagimburg hi. Of course! Just make it sure that it's working before pr please, because I have no possibility to test your changes. Thanks!

**Comment by dimagimburg** [comment]
Yes sir. It could take me some time to update the code it though, because I am doing it on my spare time while reading the book. Do you want me to pull-request for each chapter or pull-request for a bunch of chapters together? (it would be easier to me to pull-request chapter by chapter). Thanks for opening this repo BTW, it helped me a lot, hope it would help other people as well.

**Comment by ales-tsurko** [comment]
Do as you like. From my side it's quite simple to merge your requests.
You're welcome! Thank you for the contribution!
